torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1676,6 +1676,38 @@ def sample_generator():
 
     return list(sample_generator())
 
+def sample_inputs_copysign(op_info, device, dtype, requires_grad):
+    def _make_case(case):
+        if isinstance(case, tuple):
+            return make_tensor(case, device, dtype,
+                               low=None, high=None,
+                               requires_grad=requires_grad)
+        else:
+            return case
```

Comment:
There isn't really any non-tuple case for you to hit in the examples below, right?

@mruberry is there a more well known function that's supposed to be used in this case?

Reply:
@ezyang The non-tuple cases are hit when constructing args of SampleInput, please kindly refer to https://github.com/pytorch/pytorch/pull/54945/files#r605009953.
I think this branch of checking tuple causes this ambiguity.
It's nearly midnight in my timezone and it took some hours to build PyTorch from the source, I will try improving the readability of this PR tomorrow.